x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/description_step/helpers.tsx

```diff
@@ -69,6 +69,27 @@ const Query = styled.div`
   white-space: pre-wrap;
 `;
 
+export const getQueryLabel = (ruleType: Type | undefined): string => {
+  switch (ruleType) {
+    case 'query':
+      return i18n.QUERY_LABEL;
+    case 'eql':
+      return i18n.EQL_QUERY_LABEL;
+    case 'saved_query':
+      return i18n.SAVED_QUERY_LABEL;
+    case 'threshold':
+      return i18n.QUERY_LABEL;
+    case 'threat_match':
+      return i18n.THREAT_QUERY_LABEL;
```

Comment:
this one still should be custom query label

<img width="1166" alt="Screenshot 2024-03-18 at 11 04 11" src="https://github.com/elastic/kibana/assets/92328789/07ee6b0e-fe9f-407a-b1ad-9dc54ee1d33f">
<img width="1019" alt="Screenshot 2024-03-18 at 11 04 46" src="https://github.com/elastic/kibana/assets/92328789/e57ee99b-aaca-4ceb-915b-324c20f73f56">